pkg/venafi/cloud/connector.go

```diff
@@ -606,12 +606,6 @@ func (c *Connector) RevokeCertificate(revReq *certificate.RevocationRequest) (en
 			return nil, fmt.Errorf("failed to find CA account %q", revReq.CertificateAuthorityAccountName)
 		}
 
-		//validating the CAType is into the set of supported types to perform revocation
-		if !CATypesSupportedForRevocation[caAccount.CertificateAuthorityType] {
```

Comment:
@angelmoo could you please remove obsolete consts also, i.e CATypesSupportedForRevocationSlice and CATypesSupportedForRevocation?

Reply:
they are removed now